docs/schema.md

```diff
@@ -241,6 +241,29 @@ An `element` must contain the structure with the same rules as an attribute. As
 }
 ```
 
+#### `Extension`<a name="schema-entitytypes-shape-extension"></a>
+{: .no_toc }
+
+For attributes of `"type": "Extension"`, you must also specify the `name` of the specific extension type.
+There are two extension types: `ipaddr` for IP address values, and `decimal` for decimal values.
```

Comment:
Should this say "currently" to indicate we are open to/may add more?

Reply:
I don't think "currently" is necessary. Everything is ultimately subject to change. :)

Reply:
Agreed - not necessary!